apps/jupyter/run-jupyter.sh

```diff
@@ -25,6 +27,9 @@ rm /app/.local/share/jupyter/runtime/jpserver-*-open.html 2>/dev/null
 if [[ ${SSL} == 0 ]]; then
     echo "Jupyter not serving over SSL."
     SSLARG=
+elif [[ -f ${ARYN_DIR}/hostcert.pem && -f ${ARYN_DIR}/hostkey.pem ]]; then
+    echo "Using SSL certificate from ${ARYN_DIR}"
+    SSLARG="--certfile=\"${ARYN_DIR}/hostcert.pem\" --keyfile=\"${ARYN_DIR}/hostkey.pem\""
```

Comment:
I would do this as SSLARG=("--certfile=\"${ARYN_DIR}/hostcert.pem\"" "--keyfile=\"${ARYN_DIR}/hostkey.pem\"")
and then use it as "${SSLARG[@]}"
That guarantees it will work even if there are spaces in ARYN_DIR.
Near as I can tell as written it will split the spaces:
```
% A="--c=\"abc def ghi\" --d=\"a b c d e\""
% cat /tmp/foo.sh 
echo $#
echo 1:$1
echo 2:$2
echo 3:$3
echo 4:$4
% /tmp/foo.sh $A
8
1:--c="abc
2:def
3:ghi"
4:--d="a
```